### PR TITLE
Better profile page

### PR DESCRIPTION
**Pull request by jchen**
Better profile page.

### CODE REVIEW

File: BLT/BLT/UserProfileViewController.swift [hunk 1]
```diff
@@ -56,30 +66,54 @@ class UserProfileViewController: UIViewController
         userImage.clipsToBounds = true
         userImage.layer.cornerRadius = 50
         
-        userDataChart.clipsToBounds = true
-        userDataChart.layer.cornerRadius = 25
-        userDataChart.backgroundColor = .gray
+//        userDataChart.clipsToBounds = true
```
Comment (EKKOING) [inline BLT/BLT/UserProfileViewController.swift]: Remove comments for code that has been replaced

**Review by EKKOING — CHANGES_REQUESTED** [review]
Need to remove commented out code that has been replaced

File: BLT/BLT/UserProfileViewController.swift (hunk 1 above)
Comment (jchen) [reply BLT/BLT/UserProfileViewController.swift]: Fixed in commit 3155623.

**Review by EKKOING — APPROVED** [review]
Looks Really Nice

**Pull request merged**